pkg/controller/controller.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Crossplane Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+// Args args used by controller
+type Args struct {
+	// RevisionLimit is the maximum number of revisions that will be maintained.
+	// The default value is 10.
```

Comment:
Make the default number be 50 will be better I think.

Reply:
👌